experimental/add2app/SimpleIOSExample/Podfile

```diff
@@ -0,0 +1,26 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+# Added these two lines for Flutter:
```

Comment:
Not sure what this comment meant.

Reply:
It makes some sense in the context of the add2app guide, but you're right that it looks odd without it. I've updated it.